anvil/core/src/eth/mod.rs

```diff
@@ -263,6 +263,14 @@ pub enum EthRequest {
         #[cfg_attr(feature = "serde", serde(default))] GethDebugTracingOptions,
     ),
 
+    /// geth's `debug_traceCall`  endpoint
+    #[cfg_attr(feature = "serde", serde(rename = "debug_traceCall"))]
+    DebugTraceCall(
+        EthTransactionRequest,
+        #[cfg_attr(feature = "serde", serde(default))] Option<BlockId>,
+        #[cfg_attr(feature = "serde", serde(default))] GethDebugTracingOptions,
+    ),
```

Comment:
great, let's add a smol sanity test for this call, see end of file for reference